src/runtime/auto-prompt-manager.ts

```diff
@@ -1117,8 +1117,14 @@ export class AutoPromptManager {
     } else if (actionType === TYPE_REWARDED_SURVEY) {
       return isSurveyEligible;
     } else if (actionType === TYPE_REWARDED_ADS) {
-      // Because we have fetched the article endpoint googletag.cmd should already exist.
-      const googletagExists = !!this.deps_.win().googletag?.cmd;
+      /*
+       * Because we have fetched the article endpoint gpt.js should
+       * already be ready, but this is a race condition. Failing the race
+       * condition results in not showing the prompt.
+       */
+      const googletagExists =
+        !!this.deps_.win().googletag?.apiReady &&
```

Comment:
I guess this change will go away once you sync to head?

Reply:
Yes, will reconfirm everything looks correct after syncing.